src/train_cmd.cpp

```diff
@@ -1973,6 +1973,8 @@ static bool IsWholeTrainInsideDepot(const Train *v)
  */
 void ReverseTrainDirection(Train *v)
 {
+	//DEBUG(driver, 1, "Reversing train %d on tile %d", v->index, v->tile);
```

Comment:
Please remove the commented-out debug statement, we like to keep only clean, working code in OpenTTD.